Lib/test/test_future_stmt/test_future.py

```diff
@@ -25,57 +25,71 @@ def check_syntax_error(self, err, basename, lineno, offset=1):
         self.assertEqual(err.offset, offset)
 
     def test_future1(self):
-        with import_helper.CleanImport('future_test1'):
-            from test import future_test1
+        with import_helper.CleanImport('test.test_future_stmt.future_test1'):
+            from test.test_future_stmt import future_test1
             self.assertEqual(future_test1.result, 6)
 
     def test_future2(self):
-        with import_helper.CleanImport('future_test2'):
-            from test import future_test2
+        with import_helper.CleanImport('test.test_future_stmt.future_test2'):
+            from test.test_future_stmt import future_test2
             self.assertEqual(future_test2.result, 6)
 
-    def test_future3(self):
-        with import_helper.CleanImport('test_future3'):
-            from test import test_future3
+    def test_future_single_import(self):
+        with import_helper.CleanImport(
+            'test.test_future_stmt.test_future_single_import',
+        ):
+            from test.test_future_stmt import test_future_single_import
+
+    def test_future_multiple_imports(self):
+        with import_helper.CleanImport(
+            'test.test_future_stmt.test_future_multiple_imports',
+        ):
+            from test.test_future_stmt import test_future_multiple_imports
+
+    def test_future_multiple_features(self):
+        with import_helper.CleanImport(
+            "test.test_future_stmt.test_future_multiple_features",
+        ):
+            from test.test_future_stmt import test_future_multiple_features
 
     def test_badfuture3(self):
         with self.assertRaises(SyntaxError) as cm:
-            from test import badsyntax_future3
+            from test.test_future_stmt import badsyntax_future3
         self.check_syntax_error(cm.exception, "badsyntax_future3", 3)
 
     def test_badfuture4(self):
         with self.assertRaises(SyntaxError) as cm:
-            from test import badsyntax_future4
+            from test.test_future_stmt import badsyntax_future4
         self.check_syntax_error(cm.exception, "badsyntax_future4", 3)
 
     def test_badfuture5(self):
         with self.assertRaises(SyntaxError) as cm:
-            from test import badsyntax_future5
+            from test.test_future_stmt import badsyntax_future5
         self.check_syntax_error(cm.exception, "badsyntax_future5", 4)
 
     def test_badfuture6(self):
         with self.assertRaises(SyntaxError) as cm:
-            from test import badsyntax_future6
+            from test.test_future_stmt import badsyntax_future6
         self.check_syntax_error(cm.exception, "badsyntax_future6", 3)
 
     def test_badfuture7(self):
```

Comment:
... Maybe these tests would be more explicit if they would not use a file, but pass the code as a string (maybe use textwrap.dedent()).